offlineimap/utils/distro.py

```diff
@@ -43,13 +44,13 @@ def get_os_name():
     if OS.startswith('linux'):
         DISTRO = platform.linux_distribution()[0]
         if DISTRO:
-          OS = OS + "-%s" % DISTRO.lower()
+          OS = OS + "-%s" % DISTRO.split()[0].lower()
```

Comment:
AFAICT, this is not applicable for all the possible values of `platform.linux_distribution()`. Why not use `'linux-gentoo base system'` as keyword instead?